src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.js

```diff
@@ -68,8 +64,64 @@ const ReportsTrafficSources = function({ domainName, dateFrom, dependencies: { d
                     </span>
                     )
                   </span>
-                </TrafficSourceHeader>
-              </TrafficSourceContainer>
+                </S.ListItemHeader>
+                {trafficSource.withEmail || trafficSource.withEmail === 0 ? (
```

Comment:
What happens if all traffic sources are without email? This section will be hidden, is it ok?

Reply:
don't because i add the validation `trafficSource.withEmail === 0` have 0 value and show